apps/trading/client-pages/market/trade-panels.tsx

```diff
@@ -211,7 +211,7 @@ const useViewLabel = (view: TradingView) => {
     activeOrders: t('Open'),
     inactiveOrders: t('Order history'),
     stopOrders: t('Advanced orders'),
-    collateral: t('Collateral'),
+    assets: t('Assets'),
```

Comment:
Wondering if we want to go ahead and add the asset management UI to the new BottomDrawer. Will get back to you